privacyIDEA-impl/src/main/java/org/privacyidea/action/ChallengeResponseAction.java

```diff
@@ -176,7 +191,12 @@ protected void extractChallengeData(@Nonnull PIResponse piResponse)
         {
             if ("poll".equals(c.getClientMode()))
             {
-                piFormContext.setImagePush(c.getImage());
+                if (StringUtil.isNotBlank(c.getImage()))
+                {
+                    piFormContext.setImagePush(c.getImage());
+                    //todo Walkaround to show the push image by enrollment via challenge. Waiting for an update of privacyIDEA response.
```

Comment:
workaround not walkaround :D

Reply:
maybe clarify more: "waiting for updating of privacyidea that clearly indicates enroll_via_multichallenge challenges."

Reply:
[ce56f39](https://github.com/privacyidea/shibboleth-plugin/pull/71/commits/ce56f3991dba0003147d337a14219e8d95a617f3)